### DISCUSSION TITLE
Why `'$app/sth'`s have to look like this?

### DISCUSSION THREAD

**Discussion by ehzro3** [sveltejs/kit#5819] [Ideas]
Can't we have everything from
- [$app/env](https://kit.svelte.dev/docs/modules#$app-env)
- [$app/navigation](https://kit.svelte.dev/docs/modules#$app-navigation)
- [$app/paths](https://kit.svelte.dev/docs/modules#$app-paths)
- [$app/stores](https://kit.svelte.dev/docs/modules#$app-stores)

just in `'$kit'`?

**Comment by Mlocik97** [comment]
I think `$app/` makes more sense... `$kit/` for me would evoke it's some internal module of kit, or resp. some module that is "deep inside kit", or that it contains helpers, but not something "app" related. And I find all of those more "app" related, than Kit related.

**Reply by ehzro3** [reply]
Thanks for reply! I might not understand larger context of node development but for me `$app` sounds like the end product I build. And `$kit` is where I take my tools out to build it.

**Reply by jaclas** [reply]
But $app points to the internals of app, not the kit :-)
